ui/src/App.tsx

```diff
@@ -85,15 +130,7 @@ function App() {
             },
           })}
         >
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
-          <Box></Box>
+          {grid_boxes}
```

Comment:
```suggestion
          {[...Array(9).keys()].map(entry => <SquareButton key={entry} />)}
```

Reply:
Applied this change but using GridButton instead of SquareButton